bootstrap.lua

```diff
@@ -0,0 +1,43 @@
+-- Easy installer. Bootstrapped by http://pastebin.com/p8PJVxC4
+
+local base_url, dest = select(1,...)
+dest = dest or 'computercraft-github'
```

Comment:
What your essentially creating here is a custom rom... I think the best default would be `github.rom`.